struqture-py-macros/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "struqture-py-macros"
-version = "1.4.1"
+version = "1.4.2"
```

Comment:
This would be a 1.4 to 1.5 version bump according to semver, since we are adding new functionality.